lambdas/handlers/delete_document_reference_handler.py

```diff
@@ -47,13 +48,15 @@ def lambda_handler(event, context):
                 404, "No documents available", "DELETE"
             ).create_api_gateway_response()
 
-        document_service.delete_documents(table, results)
+        document_service.delete_documents(
+            table_name=table,
+            document_references=results,
+            type_of_delete=str(S3LifecycleTags.SOFT_DELETE.value),
+        )
     except ClientError as e:
         logger.info(str(e))
         return ApiGatewayResponse(
             500, "Failed to delete documents", "DELETE"
         ).create_api_gateway_response()
 
-    return ApiGatewayResponse(
-        204, "Successfully deleted documents", "DELETE"
-    ).create_api_gateway_response()
+    return ApiGatewayResponse(200, "Success", "DELETE").create_api_gateway_response()
```

Comment:
do we need to amend front end to check for 200 rather than 204?  
(ignore me if it is already done or planned in other ticket)

Reply:
We changed it so it could be consumed correctly in the FE, Rachel is aware and has made this change in the FE :)